api/plugin_helpers.go

```diff
@@ -62,6 +63,7 @@ func (f *PluginAPIClientMeta) FlagSet() *flag.FlagSet {
 	fs.StringVar(&f.flagCAPath, "ca-path", "", "")
 	fs.StringVar(&f.flagClientCert, "client-cert", "", "")
 	fs.StringVar(&f.flagClientKey, "client-key", "", "")
+	fs.StringVar(&f.flagServerName, "tls-server-name", "", "")
```

Comment:
Should we update any documentation for this?

Reply:
I had trouble finding any existing documentation that called these out. Do you happen to know if any exist?

Reply:
I didn't find anything either